Functions.Templates/Templates/EventHubTrigger-FSharp-Isolated/.template.config/template.json

```diff
@@ -47,7 +47,7 @@
       "ManualInstructions": [],
       "args": {
         "referenceType": "package",
-        "reference": "Microsoft.Azure.Functions.Worker.Extensions.EventHubs", "version": "4.2.0",
+        "reference": "Microsoft.Azure.Functions.Worker.Extensions.EventHubs", "version": "4.3.0",
```

Comment:
Is this the correct major version?

Reply:
I just aligned that with the C# template.   Should we bump them both up?

Reply:
God... yes.   That's from 2021.